stable/chaoskube/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: chaoskube
-version: 0.6.1
+version: 0.6.1-1
```

Comment:
Does anyone have a better approach? `v0.6.2` would be a bit confusing at this point.

Reply:
Why would 0.6.2 be confusing for a chart version? Charts are versioned separately from app versions. Adding the `-1` on the end notes that it's a prerelease to `0.6.1`. Helm expects semver.

Reply:
@mattfarina Thanks for the hint. i changed it to 0.6.2.